### ISSUE TITLE
Ethereum Ropsten read limit exceeded

### ISSUE THREAD

**Issue by akme** [trezor/blockbook#606]
Blockbook can't sync Ethereum Ropsten Testnet. Backend synced, but blockbook gives an error.

Backend version: Geth/v1.10.2-stable-97d11b01/linux-amd64/go1.16
Blockbook Version / Commit / Build | 0.3.4 / e046e3a / 2021-04-21T15:39:05+00:00
```
E0423 07:59:27.875632       1 worker.go:1744] GetChainInfo error websocket: read limit exceeded
E0423 07:59:27.875718       1 sync.go:335] getBlockWorker 6 connect block error hash 0xc02b5459997699df8b7a2bc45db78d784b1938b360ac3bc9f92764ee290e4ea5, height 599281: websocket: read limit exceeded. Retrying...
E0423 07:59:28.487353       1 sync.go:335] getBlockWorker 3 connect block error hash 0xd092253c94e5f24426fff2c9a203ec921b2df0d188d0f7ffb654be0dbec15ddd, height 599275: websocket: read limit exceeded. Retrying...
```

**Comment by akme** [comment]
Switched rpc_url from websocket to http and it started to sync further

**Comment by martinboehm** [comment]
It is already fixed by this commit 0bf1d17edf0f4e99528f03b7327e747949f44bc7, although not yet merged to master
